testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -573,34 +573,32 @@ def test_sklearn_affinity_propagation(self, ens1):
                               "clusters: {0} {1}".format(len(cc1), len(cc2))
 
     def test_ClusterCollection_init(self, cc):
-        if np.testing.assert_allclose(cc.clusters[0].elements , [0,1,2])and \
-               np.testing.assert_allclose(cc.clusters[1].elements , [3, 4]) \
-                   and \
-               np.testing.assert_allclose(cc.clusters[2].elements , [5, 6, 7])\
-                   and \
-               cc.clusters[0].centroid == 1 and \
-               cc.clusters[1].centroid == 3 and \
-               cc.clusters[2].centroid == 5: \
-                      print("ClusterCollection was not constructed correctly")
+        assert_equal(cc.clusters[0].elements , [0,1,2]) and \
+        assert_equal(cc.clusters[1].elements , [3, 4]) and \
+        assert_equal(cc.clusters[2].elements , [5, 8, 7]) and \
+        assert_equal(cc.clusters[0].centroid , 1) and \
+        assert_equal(cc.clusters[1].centroid , 3) and \
+        assert_equal(cc.clusters[2].centroid , 5), \
+                   "ClusterCollection was not constructed correctly"
 
     def test_Cluster_init(self, cluster):
-        if np.testing.assert_allclose(cluster.elements,[0, 1, 2]) and \
-               cluster.centroid == 1: \
-                      print("Cluster was not constructed correctly")
+        assert_equal(cluster.elements,[0, 1, 2]) and \
+              assert_equal(cluster.centroid , 1), \
+                      "Cluster was not constructed correctly"
 
     def test_ClusterCollection_get_ids(self, cc):
-        assert cc.get_ids() == [0, 1, 2], \
-                     "ClusterCollection ids aren't as expected"
+        assert_equal(cc.get_ids() , [0, 1, 2], \
```

Comment:
A bit picky but PEP8 tells us

```suggestion
        assert_equal(cc.get_ids(), [0, 1, 2], \
```